Games/core/Meeting.js

```diff
@@ -735,11 +735,15 @@ module.exports = class Meeting {
         }
        
         if (!message.recipients)
-            message.recipients = this.getPlayers();
+            message.recipients = defaultRecipients || this.getPlayers();
 
         if (message.recipients.length == 0)
             return;
 
+        if (defaultRecipients) {

```

Comment:
im not sure why this is needed

Reply:
I also wasn't sure, but I think its may be due to the message recipients being altered by the existence of defaultRecipients. Could be a formality in a way.